spec/lib/va_profile/prefill/military_information_spec.rb

```diff
@@ -8,365 +8,299 @@
+        VCR.use_cassette('va_profile/military_personnel/post_read_service_history_200',
+                         match_requests_on: %i[method body]) do
+          expect(subject.sw_asia_combat).to eq(false)
         end
       end
 
-      describe '#compensable_va_service_connected' do
-        it 'returns false if the rating percentage is not considered "low"' do
-          VCR.use_cassette('va_profile/disability/disability_rating_200_high_disability') do
-            response = subject.compensable_va_service_connected
-
-            expect(response).to eq(false)
-          end
-        end
-      end
-
-      describe '#va_compensation_type' do
-        it "returns 'highDisability' when veteran is paid for a highDisbility" do
-          VCR.use_cassette('va_profile/disability/disability_rating_200_high_disability') do
-            response = subject.va_compensation_type
-
-            expect(response).to eq('highDisability')
-          end
+      it 'returns false if there is no deployment location' do
+        VCR.use_cassette('va_profile/military_personnel/service_history_200_many_episodes') do
+          expect(subject.sw_asia_combat).to eq(false)
         end
       end
     end
 
-    context 'using bio path disabilityRating, LOW PERCENTAGE' do
-      describe '#is_va_service_connected' do
-        it 'returns false if veteran is paid for a disability with a low disability percentage' do
-          VCR.use_cassette('va_profile/disability/disabilityRating_200_low_disability') do
-            response = subject.is_va_service_connected
-
-            expect(response).to eq(false)
-          end
+    describe '#discharge_type' do
+      it 'returns discharge type' do
+        VCR.use_cassette('va_profile/military_personnel/post_read_service_history_200',
+                         match_requests_on: %i[method body]) do
+          expect(subject.discharge_type).to eq('general')
         end
       end
 
-      describe '#compensable_va_service_connected' do
-        it 'returns true if veteran is paid for a disability with a low disability percentage' do
-          VCR.use_cassette('va_profile/disability/disabilityRating_200_low_disability') do
-            response = subject.compensable_va_service_connected
-
-            expect(response).to eq(true)
-          end
-        end
-      end
-
-      describe '#va_compensation_type' do
-        it "returns 'lowDisability' when veteran is paid for a lowDisbility" do
-          VCR.use_cassette('va_profile/disability/disabilityRating_200_low_disability') do
-            response = subject.va_compensation_type
-
-            expect(response).to eq('lowDisability')
-          end
+      it 'with an unknown character_of_discharge_code it returns nil' do
+        VCR.use_cassette('va_profile/military_personnel/service_history_200_many_episodes') do
+          expect(subject.discharge_type).to eq(nil)
         end
       end
     end
 
-    context 'military personnel service' do
```

Comment:
😕 This file was formatted kind of oddly and then it looks like git did some weird stuff to it as well. There were two `'military personnel service'` context sections, so I deleted one. My intention was to remove the disability tests from this file, but that involved changing the indentation for the military info tests. idk if it would help to look at the file by itself and see if it makes sense: https://github.com/department-of-veterans-affairs/vets-api/blob/3f796c691a0179542205ec7be0ec16cac4713243/spec/lib/va_profile/prefill/military_information_spec.rb